isework/mega65r1.prj

```diff
@@ -15,7 +15,7 @@ vhdl work "../src/vhdl/sdcard.vhdl"
 vhdl work "../src/vhdl/ghdl_videobuffer.vhdl"
 vhdl work "../src/vhdl/ghdl_ram8x512.vhdl"
 vhdl work "../src/vhdl/ghdl_ram8x4096.vhdl"
-vhdl work "../src/vhdl/colourram.vhdl"
```

Comment:
Should this line really be deleted? Isn't it needed for the target?

Reply:
Oops, not quite sure how that happened, I'll take a look.